### DISCUSSION TITLE
What the ORP?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#941] [Q&A]
For the past few days, the reported ORP level has been dropping steadily in my pool.  It started in the low 600s, then struggled in the 500s for several days, never getting close to the 650 mV set point in spite of running the salt cell for most of the day, other than when the acid pump was running, which tends to be infrequent once  the pool has been on for a few hours.  When I tested free Cl levels with a Taylor kit, it started at 4.4 PPM and over the next few days went to 6.6 PPM.  Clearly, the ORP readings and free Cl were diverging.

Then yesterday, the probe hit the fan so to speak.  ORP readings dropped like a stone for no apparent reason as you can see in the graph. The salt cell was cranking away as a result.

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/1554e9e8-720d-4ccb-8a2d-a8450e465d94)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/7fab3d72-1539-40c2-a259-cd6fe4e47042)

This morning when the pool started, I saw some air bubbles on the probes.  I was hoping they were the source of the problem, so I drained the flow cell enough to dissipate the bubbles.  The ORP level came back up to the low 500 mV range, but then dropped again to about 110 mV after the pool was on for a few hours.  I checked the free Cl level again this morning & it was 8.6 PPM.  At that point, I stopped using ORP to control the salt cell & set it at 20% to bring the Cl level down over the next few days.

This all brings me to wonder what the ORP is going on?  I’m wondering if I’m better off dispensing with ORP entirely and just running the salt cell at a fixed rate using Nixie/REM to control pH and prevent Cl generation when acid from is being dispensed—still  an improvement over Pentair’s  approach, not to mention safer.  I’d be disappointed in considering it, but if it makes sense, so be it.

Finally, I wonder if there is a tendency for air to get into the flow cell/probes  because I tapped both the flow cell inflow and outflow lines into the filter pump.  I do think the pump pot to gets more air in it than before I plumbed the flow cell to it.  The outflow line from the flow cell feeds directly into the pump pot chamber.  Both the inflow & outflow lines to the flow cell have one-way valves, but I don’t know if they prevent air intrusion or just keep the liquids where they need to be.

I welcome theories, suggestions, & possible experiments to run to get the ORP back in the loop as the salt cell master.

Thanks for your time & thoughts!

**Comment by tagyoureit** [comment]
The waterproof SMA connectors didn't make a difference (at least not long term), huh?  I'm having a hard time coming up with any new theories here.  I can say that I never really had great luck with the consumer grade probes and had much more consistency once I went with the industrial probes.  I can't remember which you have, and don't want to suggest throwing more money at a challenging situation, but you might want to talk with the Atlas team and see if you could work something out with them.

**Reply by FlaMike** [reply]
Thank you for your response!

Sadly, all the various attempts to isolate the probe from electrical interference and/or static charges did not resolve all of my challenges.  They may have all helped, but argh!  I'm close to giving up on ORP as my friend.  I will contact A-S &  see if their techs know the magic elixir to solve my ORP problems.

BTW, I am already running the industrial probes.  They're both the "new"  Gen 3 probes with A-S installed SMA connectors.  pH has  been solid as iron, but ORP not been my BFF from day 1.

As an aside, but possibly related questions.  In your flow cell setup, do you use the Intelliflow as the inflow line to the flow cell and use a convenient PVC return line pipe for outflow or visa versa?  Can you also let me know which of the ports on your pump you're using (i.e., the pump pot port (inflow from the pool side of the pump) or the outflow (pressurized) side of the pump?  I am currently using the pump pot port--say that three times fast--for the drain side of the flow cell and the pressure side port as the inflow to the flow cell.

Thanks again for your help!

**Reply by tagyoureit** [reply]
I am using the pot port (inflow side) and the drain side of the flow cell was injecting back into the PVC after my chlorinator.  

I hope Atlas has some more magic up their sleeves for you.

**Reply by FlaMike** [reply]
Thanks for sharing your setup.  I may change mine to mimic yours & hope that less air gets into the lines and just maybe the ORP probe will behave better.  In the interim, I sent a note to Jason at Atlas this morning.  He's been helpful in the past so hopefully, he'll have some magic he can share.  I'll keep you posted!

Thanks again & have a superb weekend!
